oap-server/server-starter/src/main/resources/otel-rules/pulsar/pulsar-broker.yaml

```diff
@@ -34,66 +34,14 @@ metricPrefix: meter_pulsar_broker
 
 # Metrics Rules
 metricsRules:
-  # Cache Metrics
-  - name: cache_evictions
-    exp: pulsar_ml_cache_evictions.sum(['cluster', 'broker'])
-  - name: cache_inserted_entries
-    exp: pulsar_ml_cache_inserted_entries_total.sum(['cluster', 'broker'])
-  - name: cache_evicted_entries
-    exp: pulsar_ml_cache_evicted_entries_total.sum(['cluster', 'broker'])
-  - name: cache_entries
-    exp: pulsar_ml_cache_entries.sum(['cluster', 'broker'])
-  - name: cache_hits_rate
-    exp: pulsar_ml_cache_hits_rate.sum(['cluster', 'broker'])
-  - name: cache_hits_throughput
-    exp: pulsar_ml_cache_hits_throughput.sum(['cluster', 'broker'])
-  - name: cache_misses_rate
-    exp: pulsar_ml_cache_misses_rate.sum(['cluster', 'broker'])
-  - name: cache_misses_throughput
-    exp: pulsar_ml_cache_misses_throughput.sum(['cluster', 'broker'])
-
```

Comment:
Why are these removed?

Reply:
Pulsar only has 10 metrics with broker label.
![image](https://github.com/apache/skywalking/assets/55571188/b8c08f2c-2a33-4fa4-bff2-7dd6a429e24e)

Reply:
Then how these deleted metrics look like?

Reply:
They only have the cluster label.

Reply:
I have discussed this with your mentor. Could you sync with him? OTEL collector could add the source as an extra label.
Bookeeper should be lands a new layer and in the database catalog.

Reply:
> I have discussed this with your mentor. Could you sync with him? OTEL collector could add the source as an extra label. Bookeeper should be lands a new layer and in the database catalog.

already done.